docs/src/content/docs/ghaf/scs/slsa-framework.mdx

```diff
@@ -14,20 +16,87 @@ Supply chain Levels for Software Artifacts (SLSA) is a security framework for ta
+to bypass or forge.
 
-**Level 2** Forging the provenance or evading verification requires an explicit “attack”, though this may be easy to perform. Deters unsophisticated adversaries or those who face legal or financial risk.
+**Level 2** Forging the provenance or evading verification requires an explicit “attack”, though this may 
+be easy to perform. Deters unsophisticated adversaries or those who face legal or financial risk.
 
 In practice, this means that builds run on a hosted platform that generates and signs the provenance.
 
-**Level 3** Forging the provenance or evading verification requires exploiting a vulnerability that is beyond the capabilities of most adversaries.
+**Level 3** Forging the provenance or evading verification requires exploiting a vulnerability that is beyond
+the capabilities of most adversaries.
 
 In practice, this means that builds run on a hardened build platform that offers strong tamper protection.
 
-SLSA level is not transitive, thus level of the artifact is not dependent on the level of dependencies, which are expected to have their own SLSA levels. This makes it possible to build a Level 3 artifact from Level 0 dependencies.
+SLSA level is not transitive, thus level of the artifact is not dependent on the level of dependencies,
+which are expected to have their own SLSA levels. This makes it possible to build a Level 3 artifact
+from Level 0 dependencies.
+
+
+## Current Ghaf SLSA Status
+
+### Executive summary
+
+Ghaf operates a mixed posture: daily automated builds produce provenance and are SLSA Level 1,
+while release builds run in ephemeral, hardened environments and meet SLSA Level 3.
+Artifacts and provenance are signed (Azure Key Vault) and published with verification instructions.
+Near-term work targets HSM signing for daily builds and Nix-only ephemeral environments.
+
+### SLSA levels & scope
+
+**Daily builds** SLSA Level 1. Provenance files are generated for all builds.
+
+**Release builds** SLSA Level 3. Hosted build platform generates and signs provenance files
+for every target hardware build. Secrets are stored in Azure Key Vault. Ephemeral build
+environments are created from scratch for each release with Terraform and Nix.
+
+### Provenance & verification
+
+Provenance files are generated in Jenkins pipeline with [sbomnix](https://github.com/tiiuae/sbomnix). 
+In release builds, provenance files and target binaries are signed with an Azure Key Vault
+signing microservice. [sbomnix](https://github.com/tiiuae/sbomnix) is capable of listing
+all dependencies in the provenance file.
+
+We are making release builds for several target platforms. Release build artifacts
+(binaries, test results etc.) for each platform are stored in one tarball.
+Provenance files are always included in the tarball as well. Release tarballs are
+available for public download in [Ghaf archive](https://archive.vedenemo.dev/).
+Ghaf archive contains signature verification instructions for download verification.
```

Comment:
Maybe clarify "Ghaf archive" -> "Ghaf archive site". For a moment I were confused by the idea that this sentence talks about the ghaf tarball.